landingpage/src/components/AlterConfigForName.tsx

```diff
@@ -1,30 +1,36 @@
-import { util } from 'plebnames';
-import React, { useState } from 'react';
+import { PlebNameData, util } from 'plebnames';
+import React, { useEffect, useState } from 'react';
 import { FaCopy } from 'react-icons/fa6';
 import DropDownContent from './HeaderWithSearch/DropDownContent';
 
 /** Allowed Keys */
-type AlterOptionKey =
-	| 'nostr'
-	| 'website'
-	| 'owner'
-	| 'lightningAddress'
+export type AlterOptionKey = keyof PlebNameData
 	| 'any';
 
+
+type Output = { key: AlterOptionKey, value: string };
```

Comment:
`key` can be any string but not only `any`, so
```suggestion
type Inscription = { key: keyof PlebNameData | string, value: string };
```
or just
```suggestion
type Inscription = { key: string, value: string };
```
or just
```suggestion
type InscriptionInAscii = string
```
but then we have to validate if it is valid (e.g. "test.website=" (empty)) is not valid and copy should be prevented.
or
```suggestion
type Inscription = { inscriptionInAscii: string, valid: boolean };
```
or
```suggestion
type Inscription = { key: string, value: string|undefined };
```

Reply:
```suggestion
type Inscription = { key: AlterOptionKey|string, inscriptionInAscii: string, valid: boolean };
```